dowhy/causal_estimator.py

```diff
@@ -215,6 +313,9 @@ def __init__(self, estimate, target_estimand, realized_estimand_expr, **kwargs):
     def add_significance_test_results(self, test_results):
         self.significance_test = test_results
 
+    def add_confidence_intervals(self, confidence_intervals):
+        self.confidence_intervals = confidence_intervals
```

Comment:
isn't it a single confidence_interval? Not sure if the plural "intervals" is correct here.